packages/medusa/src/models/cart-sales-channel.ts

```diff
@@ -0,0 +1,27 @@
+import { BeforeInsert, Column, Index } from "typeorm"
+import { MedusaV2Flag } from "@medusajs/utils"
+
+import { generateEntityId } from "../utils"
+import { SoftDeletableEntity } from "../interfaces"
+import { FeatureFlagEntity } from "../utils/feature-flag-decorators"
+
+@FeatureFlagEntity([MedusaV2Flag.key, "sales_channels"])
+export class CartSalesChannel extends SoftDeletableEntity {
+  @Index("cart_sales_channel_cart_id_unique", {
+    unique: true,
+  })
+  @Column()
+  cart_id: string
+
+  @Column()
+  sales_channel_id: string
+
+  /**
+   * @apiIgnore
+   */
+  // NOTE: not working
```

Comment:
question: Should we act on this?

Reply:
This is just a warning since there are some cases when this callback is not triggered but we have a workaround for that so I'm not sure if there is anything actionable ATM on our end.

Reply:
Got it, can we include that in the comment, so it is more clear for someone that does not have the context?

Reply:
sure!